cript_graph_json/JSON/cao_protein.json

```diff
@@ -1,2483 +1,5572 @@
+                }
+              ]
+            },
+            {
+              "node": [
+                "Process"
+              ],
+              "name": "SPI Centrifugation",
+              "type": "multistep",
+              "ingredient": [
+                {
+                  "node": [
+                    "Ingredient"
+                  ],
+                  "material": {
+                    "node": [
+                      "Material"
+                    ],
+                    "name": "Soy protein isolate",
+                    "uid": "_:Soy protein isolate"
+                  },
+                  "quantity": [
+                    "null"
+                  ]
+                },
+                {
+                  "node": [
+                    "Ingredient"
+                  ],
+                  "material": {
+                    "node": [
+                      "Material"
+                    ],
+                    "name": "water",
+                    "uid": "_:water"
+                  },
+                  "quantity": [
+                    "null"
+                  ]
+                }
+              ],
+              "product": [
+                {
+                  "node": [
+                    "Material"
+                  ],
+                  "name": "SPI supernatant",
+                  "uid": "_:SPI supernatant"
+                }
+              ]
+            },
+            {
+              "node": [
+                "Process"
+              ],
+              "name": "PPI Centrifugation",
+              "type": "multistep",
+              "ingredient": [
+                {
+                  "node": [
+                    "Ingredient"
+                  ],
+                  "material": {
+                    "node": [
+                      "Material"
+                    ],
+                    "name": "Pea protein isolate",
+                    "uid": "_:Pea protein isolate"
+                  },
+                  "quantity": [
+                    "null"
+                  ]
+                },
+                {
+                  "node": [
+                    "Ingredient"
+                  ],
+                  "material": {
+                    "node": [
+                      "Material"
+                    ],
+                    "name": "water",
+                    "uid": "_:water"
+                  },
+                  "quantity": [
+                    "null"
+                  ]
+                }
+              ],
+              "product": [
+                {
+                  "node": [
+                    "Material"
+                  ],
+                  "name": "PPI supernatant",
+                  "uid": "_:PPI supernatant"
+                }
+              ]
+            },
+            {
+              "node": [
+                "Process"
+              ],
+              "name": "WPI Methacrylation",
+              "type": "multistep",
+              "prerequisite_process": [
```

Comment:
Why not referencing it?

Reply:
because I was not sure how to.

how would I reference it instead?

Reply:
> how would I reference it instead or what the rules are for that?

Yes, see my review comment above.